torch/csrc/jit/tracer.h

```diff
@@ -74,6 +74,23 @@ inline Value* getValueTrace(const Variable& var) {
   return it->second;
 }
 
+inline Value* getNestedValueTrace(const std::shared_ptr<TracingState>& state, const IValue &v) {
```

Comment:
Do we need to explicitly pass in `state` here? Can we just call `getTracingState()` like in `getValueTrace`?